__tests__/node/utils/deeplyParseHeader.spec.ts

```diff
@@ -1,34 +0,0 @@
-import { test, expect } from 'vitest'
-import { deeplyParseHeader } from 'node/utils/parseHeader'
-
-test('deeplyParseHeader', () => {
-  const asserts: Record<string, string> = {
```

Comment:
You probably can copy these test cases (without leading # and spaces in outcome) to https://github.com/mdit-vue/mdit-vue/blob/main/packages/plugin-title/tests/title-plugin.spec.ts

Reply:
Yeah we could. But IMO we don't need to worry about them because all the markdown syntax has already been parsed correctly by markdown-it 😄 We can add some cases once when found any unexpected behaviors.